src/Audio/Effects/UnstableEffect.cs

```diff
@@ -0,0 +1,14 @@
+﻿using OpenTK.Audio.OpenAL;
+
+namespace RPVoiceChat.Audio.Effects
+{
+    public class UnstableEffect : SoundEffect
+    {
+        public UnstableEffect(int source) : base(source) { }
+
+        protected override int GenerateEffect()
+        {
+            return (int)EffectType.Distortion;
```

Comment:
`SoundEffect.effect` is an internal ID of the OpenAL effect, you get it when you generate new OpenAl effect with `ALC.EFX.GenEffect()`
`UnstableEffect` doesn't use any effects as of now so we can just throw `NotImplementedException`

Reply:
Done 👍